content/sql/concepts/dates/terms/date-trunc/date-trunc.md

```diff
@@ -0,0 +1,95 @@
+---
+Title: 'DATE_TRUNC()'
+Description: 'Function to truncate a date, time, or timestamp to a specified interval (e.g. day or week). Available in PostgreSQL.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Database'
+  - 'Date'
+  - 'Queries'
+  - 'PostgreSQL'
+CatalogContent:
+  - 'learn-sql'
+  - 'paths/analyze-data-with-sql'
+  - 'paths/design-databases-with-postgresql'
+---
+
+The **`DATE_TRUNC()`** function in PostgreSQL is used to truncate a date, time, or timestamp to a specified interval, such as the day, week, or month.
+
+The `DATE_TRUNC()` function is particularly useful for time series analysis, where you want to understand how a value changes over time. Practical examples would include analyzing company's quarterly sales or determining the average hourly temperature.
+
+While the `DATE_TRUNC()` syntax is specific to PostgreSQL, SQL Server's `DATETRUNC()` function and MySQL's `EXTRACT()` function offer similar functionality.
```

Comment:
Hi @SSwiniarski - for what it's worth, it's also available in SQL Server as datetrunc(), so could extend the entry to include that variation similar how [date_add](https://www.codecademy.com/resources/docs/sql/dates/dateadd) shows SQL Server + My SQL syntax. But also totally fine if you would rather not include it - just let me know.

Reply:
@mbcrosier Can we make a note at the top, right below the intro, with the relevant info on where it's available? cc: @SSwiniarski

Reply:
hi @yangc95 - Sorry for the delay. Just moved the language details to the top and added a SQL Server example